web/next.config.js

```diff
@@ -201,127 +278,1679 @@ const nextConfig = {
+				permanent: true,
+			},
+			{
+				source: "/about/what-we-do/life-sciences",
+				destination:
+					"/what-nice-does/life-sciences-how-to-get-your-product-to-market",
+				permanent: true,
+			},
+			{
+				source: "/forms/life-sciences-contact-us",
+				destination: "/forms/nice-advice-contact-us",
+				permanent: true,
+			},
+			{
+				source: "/standards-and-indicators/index",
+				destination: "/indicators/published",
+				permanent: true,
+			},
+			{
+				source: "/about/who-we-are/our-charter",
+				destination: "/about-us/corporate-publications/our-charter",
+				permanent: true,
+			},
+			{
+				source: "/Get-Involved/our-committees",
+				destination: "/get-involved/our-committees",
+				permanent: true,
+			},
+			{
+				source: "/Get-Involved/Consultations",
+				destination: "/get-involved/consultations",
+				permanent: true,
+			},
+			{
+				source: "/standards-and-indicators",
+				destination: "/what-nice-does/standards-and-indicators",
+				permanent: true,
+			},
+			{
+				source: "/get-involved/contact-us",
+				destination: "/contact-us",
+				permanent: true,
+			},
+			{
+				source: "/about/who-we-are/board",
+				destination: "/about-us/our-board/board-members",
+				permanent: true,
+			},
+			{
+				source: "/get-involved/careers",
+				destination: "/careers",
+				permanent: true,
+			},
+			{
+				source: "/news/press-and-media",
+				destination: "/contact-us",
+				permanent: true,
+			},
+			{
+				source: "/re-using-our-content",
+				destination: "/reusing-our-content",
+				permanent: true,
+			},
+			{
+				source: "/covid-19/rapid-c19",
+				destination: "/rapid-c-19",
+				permanent: true,
+			},
+			{
+				source: "/get-involved/jobs",
+				destination: "/careers",
+				permanent: true,
+			},
+			{
+				source: "/guidance/proposed",
+				destination: "/guidance/awaiting-development",
+				permanent: true,
+			},
+			{
+				source: "/about/what-we-do",
+				destination: "/what-nice-does",
+				permanent: true,
+			},
+			{
+				source: "/bnfc-uk-only",
+				destination: "/bnfc-via-nice-is-only-available-in-the-uk",
+				permanent: true,
+			},
+			{
+				source: "/bnf-uk-only",
+				destination: "/bnf-via-nice-is-only-available-in-the-uk",
+				permanent: true,
+			},
+			{
+				source: "/cks-uk-only",
+				destination: "/cks-is-only-available-in-the-uk",
+				permanent: true,
+			},
+			{
+				source: "/news/events",
+				destination: "/events",
+				permanent: true,
+			},
+			{
+				source: "/indicators",
+				destination: "/indicators/published",
+				permanent: true,
+			},
+			{
+				source: "/about",
+				destination: "/about-us",
+				permanent: true,
+			},
+			{
+				source: "/aac",
+				destination: "https://www.england.nhs.uk/aac/about-us/who-we-are/",
+				permanent: true,
+			},
+			{
+				source:
+					"/about/what-we-do/research-and-development/research-recommendations/:slug*",
+				destination:
+					"/about/what-we-do/science-policy-research/research-recommendations",
```

Comment:
Just spotted that this needs updating as the destination is a pre-migration URL. Confirmed with Trish that the correct destination is "/what-nice-does/our-research-work"

Reply:
Scratch that, leave as-is